src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -31,12 +31,6 @@ public partial class WorkspaceDependencyView : UserControl
 
         private IPackageInstaller packageInstaller;
 
-
-        /// <summary>
-        /// Property to check if the Dynamo session has package pending for uninstall
-        /// </summary>
-        public Boolean IsPendingRestart { get; set; }
```

Comment:
it is potentially time for us to move all this data to a model - right now it's all in the view...

Reply:
I agree, that should a better example for 3rd party devs, I checked other view extensions, all of them have similar problems